MdeModulePkg/Bus/Pci/PciHostBridgeDxe/PciHostBridge.c

```diff
@@ -548,14 +550,18 @@ InitializePciHostBridge (
                    EFI_MEMORY_UC

```

Comment:
I think you are out of date with upstream, this should have XP passed here, too, but could just be `Attributes` if the memory protection policy check is moved up.

Reply:
I think the capabilities should include EFI_MEMORY_XP regardless of if we actually set it in the following call